### PR TITLE
Doc: add Examples in scipy.optimize.fsolve docstring

### PR DESCRIPTION
**Pull request by AtsushiSakai**
#### Reference issue
Fixing a part of #7168

#### What does this implement/fix?
add Examples in scipy.optimize.fsolve docstring

### CODE REVIEW

File: scipy/optimize/minpack.py [hunk 1]
```diff
@@ -133,6 +133,20 @@ def fsolve(func, x0, args=(), fprime=None, full_output=0,
     -----
     ``fsolve`` is a wrapper around MINPACK's hybrd and hybrj algorithms.
 
+    Examples
+    --------
+    We can find the roots of a function.
```
Comment (WarrenWeckesser) [inline scipy/optimize/minpack.py:138]: This is a very generic statement, so it doesn't really add much to the example.  I suggest replacing it with something like

    Find a solution to the system of equations ``x0*cos(x1) = 4,  x1*x0 - x1 = 5``.

**Review by WarrenWeckesser — CHANGES_REQUESTED** [review]
I made one suggestion to change the first sentence of the example.  Otherwise, this looks good!

File: scipy/optimize/minpack.py (hunk 1 above)
Comment (AtsushiSakai) [reply scipy/optimize/minpack.py]: Thank you for reviewing. I fixed it.

**Review by WarrenWeckesser — APPROVED** [review]

**Comment by WarrenWeckesser** [comment]
Thanks, @AtsushiSakai, squashed and merged!

**Pull request merged**